src/mocks/data.ts

```diff
@@ -61,6 +61,17 @@ export const registrationMockUser = {
     id: 1 as const
 };
 
+export const mockUserToUpdate = {
```

Comment:
You don't need to change this, because I am aware that you were only following the pattern that was present before you joined the project. But just for your own benefit, this is awful. Problem number one is that this file is massive, which is obviously a bad thing. But the main problem is that having a reusable object like this is a bad idea. Because it's here so that it can be shared and reused, but if you change any of its values in one test, those changes will carry over other tests, which could cause them to behave unexpectedly.

Reply:
Yeah I'm not a fan of that data file at all! 

However on the topic of it being shared/reused I understood that it shouldn't cause an issue here as I don't actually change the value of it, I'm just using a shallow copy?

For example:

```setupTest("/register/student", {
      userToUpdate: {
        ...mockUserToUpdate,
        dateOfBirth: new Date("2006-03-15"),
      },
      over13CheckboxChecked: true,
    });```